Jellyfin.Plugin.OpenSubtitles/API/OpenSubtitlesController.cs

```diff
@@ -0,0 +1,75 @@
+using System;
+using System.Net.Mime;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using OpenSubtitlesHandler;
+using OpenSubtitlesHandler.Models;
+
+namespace Jellyfin.Plugin.OpenSubtitles.API
+{
+    /// <summary>
+    /// The open subtitles plugin controller.
+    /// </summary>
+    [ApiController]
+    [Produces(MediaTypeNames.Application.Json)]
```

Comment:
```suggestion
    [Produces(MediaTypeNames.Application.Json)]
    [Authorize(Policy = "DefaultAuthorization")]
```

Controller should probably require authorization

Reply:
Added in 8b64d9997bdc5a4b0eca43019f26daf0b6045052